.github/ISSUE_TEMPLATE.md

```diff
@@ -1,7 +1,8 @@
 <!---
 Verify first that your issue/request is not already reported on GitHub.
```

Comment:
Maybe add in caps
THIS FORM WILL BE READ BY A MACHINE, COMPLETE ALL SECTIONS AS DESCRIBED

Reply:
and to be clear, bcoca is not a machine, he just seems like one

Reply:
@gundalow @bcoca Better now?